userguide/snippets/alertevents.rst

```diff
@@ -136,6 +136,10 @@ Some of the conditions that trigger an alert include:
 
 * a :ref:`Cloud Sync <Cloud Sync Tasks>` task fails
 
+* a connected Uninterruptible Power Supply (UPS) switches to battery
+  power, switches to line power, communication with the UPS is lost or
+  establisted, the battery is low, or the battery needs to be replaced.
```

Comment:
This list of alert events are not complete sentences.
```suggestion
  establisted, the battery is low, or the battery needs to be replaced
```

Reply:
Is "establisted" supposed to be "established"?